sessions/Session 20, 25th of October 2022.md.md

```diff
@@ -0,0 +1,69 @@
+#session #? October 25th 2022
```

Comment:
I've been thinking of keeping the same style as for the file name/link: Session #number, Date

What are your thoughts?